src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -9,32 +9,35 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
 {
     internal class EventGenerator : IEventGenerator
     {
+        private const string TimeCreatedFormat = "MM/dd/yyyy hh:mm:ss.fff tt";
+
         public void LogFunctionTraceEvent(TraceLevel level, string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, Exception exception = null)
         {
+            string timeCreated = DateTime.Now.ToString(TimeCreatedFormat);
```

Comment:
I know we're using `tt` in the format, but is there a reason we want to be timezone sensitive here as opposed to using `UtcNow`?